Cubical/Categories/Displayed/Limits/Cartesian.agda

```diff
@@ -0,0 +1,38 @@
+{-# OPTIONS --safe #-}
+-- Note: Max wrote this
```

Comment:
You can just put `Co-authored-by: Max New <maxsnew@gmail.com>` in the commit message. See https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors#creating-co-authored-commits-on-the-command-line